Subject: Add shadow as target to top-level Makefile
 Now that src/auto/config.mk isn't always present, "make -C src shadow"
 breaks.  Adding the target to the top-level Makefile allows one to use "make
 shadow" directly, which will ensure src/config.mk.dist is properly copied to
 src/auto/config.mk.
Author: James Vega <jamessan@debian.org>
Forwarded: no

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@
 
 # Some make programs use the last target for the $@ default; put the other
 # targets separately to always let $@ expand to "first" by default.
-all install uninstall tools config configure reconfig proto depend lint tags types test testclean clean distclean:
+all install uninstall tools shadow config configure reconfig proto depend lint tags types test testclean clean distclean:
 	@if test ! -f src/auto/config.mk; then \
 		cp src/config.mk.dist src/auto/config.mk; \
 	fi
